examples/mongodb/mongodb.go

```diff
@@ -0,0 +1,34 @@
+package mongodb
+
+import (
+	"context"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+// mongodbContainer represents the mongodb container type used in the module
+type mongodbContainer struct {
+	testcontainers.Container
+}
+
+// setupMongoDB creates an instance of the mongodb container type
+func setupMongoDB(ctx context.Context) (*mongodbContainer, error) {
```

Comment:
Were you able to take a look at the [Java implementation](https://github.com/testcontainers/testcontainers-java/blob/main/modules/mongodb/src/main/java/org/testcontainers/containers/MongoDBContainer.java), or do you think this simple example is enough?

Reply:
In any case, this LGTM. I'll merge it as is and we can iterate and evolve the example in follow ups, if you are interested

Reply:
I have a look into Java implementation, looks like it's supports replica set that looks like for me as an option. I doesn't need it, an I believe if someone will need it, I believe that it could be added as an option, as container option.
From my perspective it is good enough, as it just works and doesn't require overcomplicating it at this stage.

If we will add these rows below test, it will work, and this is most common usecase for tests, just perform operations
```go
res, err := mongoClient.Database("test").Collection("test").InsertOne(ctx, bson.M{"test": "test"})
if err != nil {
	t.Fatal(fmt.Errorf("error inserting document into mongo: %w", err))
}

if res.InsertedID == nil {
	t.Fatal("expected InsertedID to not be nil")
}
```

wdyt?

Reply:
Sounds good to me 👏